beacon-chain/rpc/validator/sync_committee.go

```diff
@@ -0,0 +1,41 @@
+package validator
+
+import (
+	"context"
+
+	ethpb "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
+	"golang.org/x/sync/errgroup"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+// GetSyncMessageBlockRoot retrieves the sync committee block root of the beacon chain.
+func (vs *Server) GetSyncMessageBlockRoot(ctx context.Context, _ *emptypb.Empty) (*ethpb.SyncMessageBlockRootResponse, error) {
+	r, err := vs.HeadFetcher.HeadRoot(ctx)
```

Comment:
Shouldn't we be enforcing the root is of block `slot -1` ? What if we get a block with root X during our slot to commit too and end up passing that root, instead of its parent